src/main/java/net/pms/encoders/VLCVideo.java

```diff
@@ -366,6 +366,10 @@ protected CodecConfig genConfig(RendererConfiguration renderer) {
 					bufSize = defaultMaxBitrates[0] / 3;
 				}
 
+				if (media.is4KVideo()) {
+					bufSize = defaultMaxBitrates[0] / 12;
+				}
```

Comment:
@Sami32 It's not 200 MB/s, it's 200 **Mb/s**, which is "only" 25 MB/s. That's fine when it comes to bandwidth alone. The file you used in your example isn't 4K, but normal HD.

I didn't know you could get 10GbE that cheap now, although it's mostly from China. Not all is what it says, but some of them would probably work. That said I'm still convinced that a decent 10 GbE switch is very expensive. Another thing is that 1x PCIe slots aren't fast enough to utilize a 10 GbE NIC, which is why these NICs are 4x or 8x (PCI 3.0 "only" has 7.877 Gb/s per lane) . 4x and 8x slots are usually only available on server motherboards, which means that for a "normal" computer you'd either need to have a free 16x port or use something like Thunderbolt.

That Netgear router was some beast of a router, but so is the price. It's not really a router in the traditional sense though, it's more like a small server with multiple 10GbE ports. Quad core 1.7 GHz CPU in a router 😆 I'd stay far away from it though, as I've had some bad experiences with the quality of Netgear equipment. Need I comment on the Apple equipment? I have both Thunderbolt and DisplayPort on my computer, but I only use the DisplayPort to connect one of my monitors. We need to stick to connectivity that can support IP for it to have any relevance for us though, and as such 10GbE seems to me like the only relevant technology.